### DISCUSSION TITLE
Set no tenant globally with ActAsTenant inside admin panel

### DISCUSSION THREAD

**Discussion by LinnJS** [avo-hq/avo#3440] [Help]
We have both Pundit and ActAsTenant in our app but for the use case of our admin dashboard we want no tenant scoping to be set.I went though the trouble of trying to add a custom resource controller but at the end get an error saying no tenant is set which is what I want.

Then if I go down the route and not setting a tenant for ever resource I get an error saying. `ActsAsTenant::Errors::NoTenantSet in Avo::Associations#index` which is also what I want. How can I do this successfully?

Error with custom ResourceController:

```
ActsAsTenant::Errors::NoTenantSet
ActsAsTenant::Errors::NoTenantSet in Avo::Associations#index
```

Which is what I want why is it erroring out? How can I fix this?

**Comment by Paul-Bob** [comment]
Hi @LinnJS have you checked this docs page? https://docs.avohq.io/3.0/guides/acts_as_tenant_integration.html

I've also found this docs section about how to not require a tenant: https://github.com/ErwinM/acts_as_tenant?tab=readme-ov-file#disabling-tenant-checking-for-a-block

And this configuration `config.require_tenant = false` https://github.com/ErwinM/acts_as_tenant?tab=readme-ov-file#configuration-options

Let me know if this helps
